thorlcr/slave/thslavemain.cpp

```diff
@@ -411,7 +411,7 @@ int main( int argc, const char *argv[]  )
             getMPServer()->setOpt(mpsopt_channelreopen, "true");
 #ifdef USE_MP_LOG
         startLogMsgParentReceiver();
-        LOG(MCdebugProgress, thorJob, "MPServer started on port %d", getFixedPort(TPORT_mp));
+        LOG(MCdebugInfo, thorJob, "MPServer started on port %d", getFixedPort(TPORT_mp));
```

Comment:
I think this category was correct.  I think progress should  correspond to logging where the timing is important (an Event), rather than whether something has happened is important.